pennylane/ops/channel.py

```diff
@@ -718,12 +717,11 @@ class QubitChannel(Channel):
             immediately pushed into the Operator queue (optional)
         id (str or None): String representing the operation (optional)
     """
-    num_params = 1
     num_wires = AnyWires
     grad_method = None
 
     def __init__(self, K_list, wires=None, do_queue=True, id=None):
-        super().__init__(K_list, wires=wires, do_queue=do_queue, id=id)
+        super().__init__(*K_list, wires=wires, do_queue=do_queue, id=id)
```

Comment:
Wait, how do we not need to change any tests if we change the signature of this operation? :o

Reply:
This class only had two extra methods, I removed the first one (`_check_batching`) because we don't need it with this new signature, and I modified the second one (`compute_kraus_matrices`) to support this change.

Reply:
Thus it makes sense that unit tests are not breaking. But it's true that some bigger integration tests might have broken with this change, but luckily that didn't happen 😄

Reply:
Ohh, I see :sweat_smile: Makes sense :)